### ISSUE TITLE
Fix price filter labels

### ISSUE THREAD

**Issue by aevanson** [PublicisSapient/accessible-ecommerce-demo#179]
Should read:

From, [price] To, [price] as aria-label

**Comment by liskulic** [comment]
@aevanson, do you have an element in mind where the aria-label should be?

**Comment by liskulic** [comment]
Confirmed on prod.
